lib/nesta/commands.rb

```diff
@@ -158,28 +158,33 @@ def initialize(*args)
           name.nil? && (raise UsageError.new('name not specified'))
           @name = name
           @gem_name = "nesta-plugin-#{name}"
+          @namespace_path = namespace_path(@gem_name)
           if File.exist?(@gem_name)
             raise RuntimeError.new("#{@gem_name} already exists")
           end
         end
 
+        def namespace_path(name)
+          name.tr('-', '/')
+        end
+
         def lib_path(*parts)
           File.join(@gem_name, 'lib', *parts)
         end
 
         def modify_required_file
-          File.open(lib_path("#{@gem_name}.rb"), 'w') do |file|
+          File.open(lib_path("#{@namespace_path}.rb"), 'w') do |file|
             file.write <<-EOF
-require "#{@gem_name}/version"
+require "#{@namespace_path}/version"
 
-Nesta::Plugin.register(__FILE__)
+Nesta::Plugin.register(@namespace_path)
             EOF
           end
         end
 
         def modify_init_file
           module_name = @name.split('-').map { |name| name.capitalize }.join('::')
-          File.open(lib_path(@gem_name, 'init.rb'), 'w') do |file|
+          File.open(lib_path(@namespace_path, 'init.rb'), 'w') do |file|
```

Comment:
I can't quite get over this, it's moronic. Quoting style is obviously something that a project is free to choose, but if there were to be a recommendation, wouldn't it make sense to choose advice that would err towards the self-documenting?

Of course it would – that was a rhetorical question.

Single quotes are great for indicating that a string doesn't require any interpolation. Anybody using doubles when they could use singles is missing a valuable trick. Hound, you're fired. You're filling my project (and my attention) with a bunch of misleading miscellany, with negative "business value".